Gemfile

```diff
@@ -5,6 +5,6 @@ source 'https://rubygems.org'
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
 # Synchronize with https://pages.github.com/versions
-ruby '>=2.7.1'
+ruby '>2.6'
```

Comment:
I'm not a ruby-connaisseur, but this allowed me to run this locally and it doesn't seem to break stuff :)